og.module

```diff
@@ -370,6 +370,12 @@ function og_og_role_insert(OgRoleInterface $role) {
     return;
   }
 
+  if ($role instanceof \Drupal\Core\Config\Entity\ConfigEntityInterface) {
```

Comment:
`OgRole` extends `Role` which, in turn, extends `ConfigEntityInterface`. This is in `og_og_rol_insert()` which is `OgRole` specific, so this check is superfluous.

Reply:
OgRoles extends Role. Thats corrent. But the type-hint of the function is OgRoleInterface. Should we change it to OgRole because the entity hook is related to the entity type and its base class.

Reply:
There is no need for it. `og_og_role_insert()` only fires when new `OgRole`s are added. So we just need to check that this is not during config sync.

Happy to stand corrected

Reply:
How about using `\Drupal::isConfigSyncing()`?